src/Fade.fs

```diff
@@ -28,4 +29,15 @@ module Fade =
         | OnExited of (unit -> unit)
         | BaseClass of string
 
-    let fade (props: FadeProps seq) (elems: ReactElement seq): ReactElement = ofImport "Fade" "reactstrap" props elems
+    let fade (props: FadeProps seq) (elems: ReactElement seq): ReactElement =
+        let customProps =
+            props
+            |> keyValueList CaseRules.LowerFirst
+
+        let typeProps =
```

Comment:
This looks like it will create the same object twice and then merge them with `Object.assign`.
I think this would suffice:
```fsharp
let fadeProps = keyValueList CaseRules.LowerFirst props
ofImport "Fade" "reactstrap" fadeProps elems
```